docs/pages/api-reference/liveblocks-react.mdx

```diff
@@ -179,14 +180,14 @@ two RoomProviders with the same room ID in your React tree.
   mutable by every client. Must either contain Live structures (e.g.
   `new LiveList()`, `new LiveObject({ a: 1 })`, etc.) or be serializable to
   JSON.
-- `shouldInitiallyConnect` (optional) - Whether or not the room should connect
-  to Liveblocks servers when the RoomProvider is rendered. By default equals to
+- `autoConnect` (optional) - Whether or not the room should immediately connect
```

Comment:
```suggestion
- `autoConnect` (optional) - Whether or not the room should automatically connect
```
If we write "immediately" people might assume there's a delay sometimes, so switching for "automatically" here.